packages/testing/index.js

```diff
@@ -1,14 +1,4 @@
-export {
-  html,
-  unsafeStatic,
-  triggerBlurFor,
-  triggerFocusFor,
-  oneEvent,
-  isIE,
-  defineCE,
-  aTimeout,
-  litFixture,
-  litFixtureSync,
-  fixture,
-  fixtureSync,
-} from '@open-wc/testing-helpers';
+import './register-chai-plugins.js';
+import './register-fixture-cleanup.js';
+
+export * from './index-no-side-effects.js';
```

Comment:
we probably want to list all the exports here as well... as otherwise, IDEs may not provide nice autocomplete... also iirc there is still a rollup bug that will break if you do it